tests/test_openmxparser.py

```diff
@@ -78,6 +78,10 @@ def test_HfO2(parser):
     assert method.electronic.smearing.width == approx(K_to_J(300))
     assert method.dft.xc_functional.correlation[0].name == 'GGA_C_PBE'
     assert method.dft.xc_functional.exchange[0].name == 'GGA_X_PBE'
+    assert (method.k_mesh.grid == [10, 10, 10]).all()
+    assert np.allclose(method.k_mesh.points[0], np.array([-0.45000, -0.45000, -0.45000]).astype(complex), rtol=0.0)
+    assert np.allclose(method.k_mesh.points[499], np.array([-0.05000, 0.45000, 0.45000]).astype(complex), rtol=0.0)
+    assert (method.k_mesh.multiplicities[0] == [2, 2, 2]).all()
```

Comment:
This looks wrong, multiplicities[0] should be float not a list, similarly in a few cases below.

Reply:
And similarly below